StreamChatSwiftUI.xcodeproj/project.pbxproj

```diff
@@ -2233,6 +2233,7 @@
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
 				DEVELOPMENT_TEAM = EHV7XZLAHA;
+				GCC_OPTIMIZATION_LEVEL = s;
```

Comment:
What's this and which target it's exactly changed?

Reply:
This is `StreamChatSwiftUITestsApp` and `StreamChatSwiftUITestsAppTests` in `Debug` configuration. This should speed up the building process a bit.